RealmSwift/Combine.swift

```diff
@@ -1001,4 +1001,137 @@ public enum Publishers {
     }
 }
 
+@available(OSX 10.15, watchOS 6.0, iOS 13.0, iOSApplicationExtension 13.0, OSXApplicationExtension 10.15, tvOS 13.0, macCatalyst 13.0, macCatalystApplicationExtension 13.0, *)
+public extension APIKeyAuth {
+    
+    /**
+     Creates a user API key that can be used to authenticate as the current user.
+     
+     @param name The name of the API key to be created.
+     */
+    func createApiKey(named: String) -> Future<UserAPIKey, Error> {
```

Comment:
@leemaguire @tgoyne Is it just me or are these methods misnamed? Should it be `createAPIKey`? Is there a reason we opted to do the acronym as lowercase?

Reply:
Using lowercase keeps it in line with the Java SDK

Reply:
Our idiomatic use of Swift/ObjC takes precedence 😅 . Can someone either open a quick PR to fix this or can it be added to this one/the CHANGELOG?